ra/ra.go

```diff
@@ -1086,3 +1086,16 @@ func (ra *RegistrationAuthorityImpl) onValidationUpdate(ctx context.Context, aut
 	ra.stats.Inc("RA.FinalizedAuthorizations", 1, 1.0)
 	return nil
 }
+
+// DeactivateAuthorization deactivates a currently valid authorization
+func (ra *RegistrationAuthorityImpl) DeactivateAuthorization(ctx context.Context, auth core.Authorization) error {
+	if auth.Status != core.StatusValid {
+		return core.MalformedRequestError("Only valid authorizations can be deactivated")
```

Comment:
I was following the same reasoning for not revoking pending authorizations in `admin-revoker` which is that a pending authz has no actual value until it is completed so there seems no reason to allow it.